src/components/ec-checkbox/ec-checkbox.story.js

```diff
@@ -45,73 +45,21 @@ stories
       return {
         checkbox1: null,
         checkbox2: null,
-      };
-    },
-    template: `
-      <div class="ec-m--24">
-        <h3> Label and Error messages coming from props</h3>
-        <ec-checkbox
-          v-if="!valueFromPropsHasError"
-          class="ec-mb--24"
-          v-model="checkbox1"
-          :disabled="valueFromPropsDisabled"
-          :label="valueFromPropsLabel">
-        </ec-checkbox>
-
-        <ec-checkbox
-          v-else="valueFromPropsHasError"
-          class="ec-mb--24"
-          v-model="checkbox1"
-          :disabled="valueFromPropsDisabled"
-          :label="valueFromPropsLabel"
-          :error-message="valueFromPropsErrorMessage">
-        </ec-checkbox>
-
-        <h3> Label and Error messages coming from template</h3>
-
-        <ec-checkbox
-          v-if="!valueFromPropsHasError"
-          v-model="checkbox2"
-          :disabled="valueFromPropsDisabled">
-          <template #label>
-            I accept the <a href="#" @click.stop.prevent="$emit('OpenTerms')"> terms and conditions </a>
-          </template>
-        </ec-checkbox>
-
-        <ec-checkbox
-          v-else="valueFromPropsHasError"
-          v-model="checkbox2"
-          :disabled="valueFromPropsDisabled">
-          <template #label>
-            I accept the <a href="#" @click.stop.prevent="$emit('OpenTerms')"> terms and conditions </a>
-          </template>
-          <template #error-message>
-            An error has occurred
-          </template>
-        </ec-checkbox>
-      </div>
-    `,
-  }))
-  .add('all', () => ({
-    components: { EcCheckbox },
-    data() {
-      return {
         errorMessage: 'An error has occurred',
         errorMessageMultiline: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Provident eos consequatur quas reiciendis aliquid ipsam ea pariatur dolorem, molestias maiores.',
       };
     },
     template: `
-      <div class="ec-m--24" style="max-width:500px;">
+      <div class="tw-m-24" style="max-width:500px;">
```

Comment:
Could you try `tw-max-w-screen-sm`? It's not exactly the same and it might affect the generated screenshots. Just give a try.

Reply:
I tested and yes, we get a slight difference between both generated screenshots, since the `tw-max-w-screen-sm` give us 76px more, and it makes that the two examples of multiline label test appear on 2 lines instead of on 3 lines. In my POV, the objective of these two tests is fulfilled anyway, since we are seeing them with a line break, so I would see right to make use of this utility class, changing the visual regression test
![image](https://user-images.githubusercontent.com/4802418/83238981-7bf4dd00-a197-11ea-9d16-2066898a508b.png)

Reply:
Finally I've included the `tw-max-w-screen-sm` class, and replaced the Lorem ipsum text by other longer, so we can maintain a very similar snapshot (with 3-line text).